src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,75 @@
+from numpy import nan
+
+
+class EstimandException(Exception):
+    pass
+
+
+RESULTS_PREFIX = "results_"
+BASELINE_PREFIX = "baseline_"
+
+
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def add_estimand_results(self, data_df, estimands, historical):
+        columns_to_return = []
+        for estimand in estimands:
+            results_col = f"{RESULTS_PREFIX}{estimand}"
+            if results_col not in data_df.columns:
+                # will raise a KeyError if a function with the same name as `estimand` doesn't exist
+                try:
+                    data_df = globals()[estimand](data_df, RESULTS_PREFIX)
+                except KeyError as e:
+                    if historical:
```

Comment:
could you just add a comment here saying why we need a results column in this case? Ie something like: In the historical case there are no results in the preprocessed data because the election has not happened yet, but when we run the model from the cli we call the LiveDataHandler to generate the set of reporting counties, and that expects a results column?

Reply:
Done!  Please check my comments and let me know if they make sense (or not 😅 ).  Thanks!